2. Building a new feature/README.md

```diff
@@ -1 +1,37 @@
-# Building a new feature
\ No newline at end of file
+# Building a new feature and making a pr
+## Overview
+When building a new feature it's worthwhile to always start on a fresh branch off of the main/master branch especially if the branch you're currently working on has had a pr created off of it. The reason is two-fold:
+
+- At Trumid, when we commit to master via pr, we strongly prefer to `squash and merge`. The `squash and merge` function coalesces all of the commits on the pr into a single, new commit and merges that into master. So if you continue working on a branch which has had a pr merged off of it, the git histories will not align: your local branch will have all the commits before the squash and merge while the master branch will only have a single commit. This may create issues and "ghost changes" in your next pr.
+- It's good to get in the habit of making sure you have the most up to date changes in your main/master branch. Creating feature branches off of updated master branches will make your life a ton easier as you'll have less merge conflicts and git issues in general.
+
+## Hands on activity
```

Comment:
i think a couple of screenshots showing the results of commands would be helpful